### PR TITLE
Include -std=c++11 when compiling protobuf if available.

### PR DESCRIPTION
**Pull request by sergiocampama**

### CODE REVIEW

**Comment by sergiocampama** [comment]
fyi @thomasvl @xfxyjwf

**Comment by xfxyjwf** [comment]
ok to test

File: m4/ax_cxx_compile_stdcxx.m4 [hunk 1]
```diff
@@ -0,0 +1,982 @@
+# ===========================================================================
```
Comment (thomasvl) [inline m4/ax_cxx_compile_stdcxx.m4]: Do you need to pull this in, or can the AC_PREREQ(2.59) line be raised to ensure this support is present?

Comment (sergiocampama) [inline m4/ax_cxx_compile_stdcxx.m4]: I'd love to, but I can't seem to find the version that makes it work, or the actual name of the macro...  :(

**Comment by xfxyjwf** [comment]
LGTM

**Pull request merged**